.azure-pipelines/Windows-CI.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       set ONNX_ML=$(onnx_ml)
       set ONNX_VERIFY_PROTO_3=$(onnx_verify_proto)
       set USE_MSVC_STATIC_RUNTIME=0
-      set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=ON -DProtobuf_USE_STATIC_LIBS=OFF -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=ON
+      set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=ON -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=ON
```

Comment:
Please also remove DONNX_USE_PROTOBUF_SHARED_LIBS=XXX in https://github.com/onnx/onnx/blob/master/.github/workflows/win_no_exception_ci.yml and https://github.com/onnx/onnx/blob/master/.github/workflows/release_win.yml

Reply:
protobuf_USE_STATIC_LIBS should be removed from those two yamls as well

Reply:
And from the build lines of protobuf in the yaml as well, since it does nothing. The flags that control how protobuf is build are ```BUILD_SHARED_LIBS``` (which controls the default of ```protobuf_BUILD_SHARED_LIBS``` and ```protobuf_BUILD_SHARED_LIBS``` itself.

Reply:
changed to use "protobuf_BUILD_SHARED_LIBS" to control the behavior. This is set to ON by default but nice to explicitly state this